src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyApprovalServiceTests.cs

```diff
@@ -256,6 +265,17 @@ await using (var context = InMemoryApplicationDbContext(contentDbContextId))
                 Assert.Equal(Immediately, updatedMethodology.PublishingStrategy);
                 Assert.True(updatedMethodology.Updated.HasValue);
                 Assert.InRange(DateTime.UtcNow.Subtract(updatedMethodology.Updated!.Value).Milliseconds, 0, 1500);
+
+                var statusList = await context
+                    .MethodologyStatus
+                    .Where(ms => ms.MethodologyVersionId == methodologyVersion.Id)
+                    .ToListAsync();
+                var status = Assert.Single(statusList);
+                Assert.Equal(methodologyVersion.Id, status.MethodologyVersionId);
+                Assert.Equal("Test approval", status.InternalReleaseNote);
+                Assert.Equal(Approved, status.ApprovalStatus);
+                Assert.InRange(DateTime.UtcNow.Subtract(status.Created!.Value).Milliseconds, 0, 1500);
+                Assert.Equal(UserId, status.CreatedById);
```

Comment:
Same comment about this being unecessary as it's already covered by `UpdateApprovalStatus_ApprovingMethodologyAddsMethodologyStatus`.

I would go through all the other variations of this test and determine whether checking the status is adding any beneft. I can see the benefit of checking in `UpdateApprovalStatus_NoStatusUpdateRequiredNoMethodologyStatusAdded` but not sure about the rest.

If they are kept, I think:

```
var status = Assert.Single(context.MethodologyStatus);
```

or this:

```
var status = await context
      .MethodologyStatus
      .Where(ms => ms.MethodologyVersionId == methodologyVersion.Id)
      .Single();
```

could be a nicer alternative to this:

```
var statusList = await context
    .MethodologyStatus
    .Where(ms => ms.MethodologyVersionId == methodologyVersion.Id)
    .ToListAsync();
var status = Assert.Single(statusList);
```